lib/util/player_configuration.js

```diff
@@ -263,6 +263,7 @@ shaka.util.PlayerConfiguration = class {
     };
 
     const lcevc = {
+      enabled: false,
```

Comment:
I think we had this enabled option in our main PR,  but i think @joeyparrish asked us to remove that as it's implied that the canvas will be used automatically if the LCEVC library is included on the page.

But i understand your intention of doing this as demo will then run everything on canvas except for drm streams.

Reply:
My intention is that it does not run unless the stream is LCEVC, if it is running all the time it is a great waste of resources that is not recommended on some TVs. On the other hand, if it is active but you do not have imported lcevc_dil.js on your website, it always launches warnings.

Reply:
Sounds good to me.